app/src/main/java/com/bocdoc/anydoc/presentation/viewholder/AiResultDetectedInfoViewHolder.kt

```diff
@@ -0,0 +1,16 @@
+package com.bocdoc.anydoc.presentation.viewholder
+
+import android.annotation.SuppressLint
+import androidx.recyclerview.widget.RecyclerView
+import com.bocdoc.anydoc.data.dto.AiResultDetectedDto
+import com.bocdoc.anydoc.databinding.ItemAiResultDetecedInfoBinding
+
+class AiResultDetectedInfoViewHolder(private val binding: ItemAiResultDetecedInfoBinding) :
+    RecyclerView.ViewHolder(binding.root) {
+
+    fun bind(detectedDto: AiResultDetectedDto) {
+        binding.tvAiResultItemDetectedInfo.text = detectedDto.detectedTitle
+        binding.tvAiResultItemDetectedDescription.text = detectedDto.detectedReason
+        binding.tvAiResultItemDetectedPercentNum.text = "${detectedDto.detectedPercent.toInt()}%"
```

Comment:
요렇게 스코프 함수인 with()를 쓰면 좋습니다!
```suggestion
        with(binding){
                tvAiResultItemDetectedInfo.text = detectedDto.detectedTitle
                tvAiResultItemDetectedDescription.text = detectedDto.detectedReason
                tvAiResultItemDetectedPercentNum.text = "${detectedDto.detectedPercent.toInt()}%"
        }
```

Reply:
오 넵! 감사합니다:)